test/config/TestConfig.cpp

```diff
@@ -447,6 +451,7 @@ TEST(Config, FleetProvisioningCli)
     ASSERT_TRUE(config.Validate());
     ASSERT_TRUE(config.fleetProvisioning.enabled);
     ASSERT_STREQ("cli-template-name", config.fleetProvisioning.templateName->c_str());
+    ASSERT_STREQ("{\"SerialNumber\": \"Device-SN\"}", config.fleetProvisioning.templateParameters->c_str());
```

Comment:
I think this class would still benefit from an additional test case that verifies that the map modified by the MapParameters function actually performs as expected in the case where a properly formatted JSON object is passed, and in the case when a malformed object is passed